src/App.tsx

```diff
@@ -261,12 +248,6 @@ function WorkerTrigger ({
       case 'setReady': setReady(params); break
       case 'setSearchResults': setSearchResults(params); break
       case 'setDidSearch': setDidSearch(params); break
-      case 'goToFrame':
-        setPlaceholderIndex(getRandomPlaceholderIndex())
```

Comment:
The removal of this makes a slight change on behavior: when on a frame page, you can see the placeholder text changing each time you click Next:

https://user-images.githubusercontent.com/722544/175797464-6f705286-ebef-4ffb-b208-b755902cebf3.mp4

But i felt this wasn't much of an important thing. Was it?
The general behavior of having a random placeholder when the page is loaded is maintained :)